src/main/java/org/zalando/nakadi/controller/EventStreamController.java

```diff
@@ -257,6 +257,8 @@ public StreamingResponseBody streamEvents(
                 final String kafkaQuotaClientId = getKafkaQuotaClientId(eventTypeName, client);
 
                 response.setStatus(HttpStatus.OK.value());
+                response.setHeader("Warning", "299 - nakadi - the Low-level API is deprecated and will " +
+                        "be removed from a future release. Please consider migrating to the High-level API.");
```

Comment:
Maybe "Subscriptions API" cause in documentation it is a preferred way of calling?

Reply:
4c1a1fb